app/styles/about.sass

```diff
@@ -1,36 +1,488 @@
+    border: 1px solid $gold
+    color: $gold
+  
+  .btn-lg
+    font-size: 18px
+    padding: 13px 35px
+    
+  // Classes
+  
+  .text-navy
+    color: $navy
+    
+  .bg-navy
+    background-color: $navy
+    color: white
+    h1, h2, h3, h4, h5, h6, a
+      color: white
+    a.btn-primary-alt
+      color: $navy
+
+
+#about-view
+  .bold
+    font-weight: bold
+  #nav-container
+    min-height: 55px
+    nav
+      background: $gold
+      &.affix
+        z-index: 1
+        position: fixed
+        top: 0
+        width: 100%
+      ul
+        padding-left: 0
+        list-style: none
+        li
+          height: 22pt
+          display: inline-block
+          margin: 10px 18px 0 18px
+          a
+            color: white
+            text-transform: uppercase
+            text-decoration: none
+        li.active
+          border-bottom: 4px solid white
+    
+  #jumbotron
+    background-color: $navy
+    background-image: url("/images/pages/about/codebackground_zoom_compressed.png")
+    background-size: cover
+    background-repeat: no-repeat
+    // height: 365px
+    
+    h1
+      color: white
+      margin-top: 100px
+      
+    h2
+      color: white
+      margin-bottom: 100px
+      
+  h3, h4
+    text-align: center
+  
+  #about-container
+    margin-top: 60px
+    background-color: white
+  
+  #mission, #community-row-2
+    margin: 50px 0
+  
+  #mission-graphic, #community-graphic
+    padding: 30px 40px
+    position: relative
+    min-height: 250px
+
+    h2
+      color: white
+      width: 50%
+    
+  #mission-graphic
+    background: $navy
+    
+    h2
+      float: right
+    
+    img
+      position: absolute
+      left: 0
+      bottom: 0
+      
+    #mission-graphic-filler
+      background: $navy
+      height: 100%
+      width: 2000px
+      position: absolute
+      right: 100%
+      top: 0
+
+  #team
+    margin-top: 150px
+    ul
+      display: flex
+      flex-wrap: wrap
+      justify-content: center
+      margin-top: 40px
+      padding: 0
+      li
+        width: 200px
+        height: 230px
+        list-style: none
+        display: inline-block
+        text-align: center
+        color: black
+        small
+          display: block
+          white-space: nowrap
+        .img-thumbnail
+          border-radius: 50%
+  #community
+    margin-top: 100px
+    
+  #community-row-1
+    display: flex
+    justify-content: center
+    #community-avatars
+      width: 90%
+      margin: 20px 5%
+
+  #community-row-2
+    margin-top: 100px
+    
+    #community-graphic
+      background: $burgandy
+
+      img
+        position: absolute
+        right: 0
+        bottom: 0
+
+      #community-graphic-filler
+        background: $burgandy
+        height: 100%
+        width: 2000px
+        position: absolute
+        left: 100%
+        top: 0
+
+  #story
+    margin-top: 150px
+    font-family: $headline-font
+    font-variant: normal
+    .media-heading, .big-text
+      //same as H1
```

Comment:
If we're going to be doing this regularly where an element has header styling but isn't a header, then we should use sass mixins or a separate class. http://sass-lang.com/guide